CHANGELOG.md

```diff
@@ -27,6 +27,12 @@ For older changes see the [archived Singularity change log](https://github.com/a
   pkg/build/types/parser.All(), `.Raw` contains the raw content of a single
   build stage. Otherwise, it is equal to `.FullRaw`.
 
+## Changes since last release
+
+### Bug fixes
+
+- Fix the issue that `APPTAINER_MESSAGELEVEL` does not have effect.
```

Comment:
Please change to
```
- Fix `$APPTAINER_MESSAGELEVEL` to correctly set the logging level.
```

Reply:
thanks for the comment! fixed in this commit
https://github.com/apptainer/apptainer/pull/1569/commits/04ffbdbffa3c88229578e5befccb5b62be3198d0